control/firstboot.xml

```diff
@@ -183,7 +183,7 @@
                 </module>
                 <module>
                     <label>Customer Center</label>
-                    <name>inst_suse_register</name>
+                    <name>registration</name>
                     <enabled config:type="boolean">false</enabled>
```

Comment:
So it is disabled by default, and the bug only surfaces when someone enables it, right?